NewHorizons/Patches/HUDPatches.cs

```diff
@@ -53,11 +58,67 @@ public static void ShipHUDMarker_OnDestroy(ShipHUDMarker __instance)
             GlobalMessenger.RemoveListener("ShipExitCloakField", __instance.RefreshOwnVisibility);
         }
 
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(ProbeHUDMarker), nameof(ProbeHUDMarker.RefreshOwnVisibility))]
+        public static bool ProbeHUDMarker_RefreshOwnVisibility(ProbeHUDMarker __instance)
+        {
+            bool insideEYE = Locator.GetEyeStateManager() != null && Locator.GetEyeStateManager().IsInsideTheEye();
```

Comment:
Does none of this NRE at the eye scene?

Reply:
It doesn't. This was just mainly code copied from mobius.